src/core/function_admin.tf

```diff
@@ -0,0 +1,399 @@
+#
+# SECRETS
+#
+
+data "azurerm_key_vault_secret" "fn_admin_ASSETS_URL" {
+  name         = "cdn-ASSETS-URL"
+  key_vault_id = data.azurerm_key_vault.common.id
+}
+
+data "azurerm_key_vault_secret" "fn_admin_AZURE_SUBSCRIPTION_ID" {
+  name         = "common-AZURE-SUBSCRIPTION-ID"
+  key_vault_id = data.azurerm_key_vault.common.id
+}
+
+data "azurerm_key_vault_secret" "adb2c_TENANT_NAME" {
+  name         = "adb2c-TENANT-NAME"
+  key_vault_id = data.azurerm_key_vault.common.id
+}
+
+data "azurerm_key_vault_secret" "devportal_CLIENT_ID" {
+  name         = "devportal-CLIENT-ID"
+  key_vault_id = data.azurerm_key_vault.common.id
+}
+
+data "azurerm_key_vault_secret" "devportal_CLIENT_SECRET" {
+  name         = "devportal-CLIENT-SECRET"
+  key_vault_id = data.azurerm_key_vault.common.id
+}
+
+data "azurerm_key_vault_secret" "adb2c_TOKEN_ATTRIBUTE_NAME" {
+  name         = "adb2c-TOKEN-ATTRIBUTE-NAME"
+  key_vault_id = data.azurerm_key_vault.common.id
+}
+
+data "azurerm_key_vault_secret" "ad_APPCLIENT_APIM_ID" {
+  name         = "ad-APPCLIENT-APIM-ID"
+  key_vault_id = data.azurerm_key_vault.common.id
+}
+
+data "azurerm_key_vault_secret" "ad_APPCLIENT_APIM_SECRET" {
+  name         = "ad-APPCLIENT-APIM-SECRET"
+  key_vault_id = data.azurerm_key_vault.common.id
+}
+
+data "azurerm_key_vault_secret" "common_AZURE_TENANT_ID" {
+  name         = "common-AZURE-TENANT-ID"
+  key_vault_id = data.azurerm_key_vault.common.id
+}
+
+data "azurerm_key_vault_secret" "apim_IO_GDPR_SERVICE_KEY" {
+  name         = "apim-IO-GDPR-SERVICE-KEY"
+  key_vault_id = data.azurerm_key_vault.common.id
+}
+
+data "azurerm_key_vault_secret" "common_SENDGRID_APIKEY" {
+  name         = "common-SENDGRID-APIKEY"
+  key_vault_id = data.azurerm_key_vault.common.id
+}
+
+# 
+# STORAGE
+#
+
+data "azurerm_storage_account" "userdatadownload" {
+  name                = "iopstuserdatadownload"
+  resource_group_name = azurerm_resource_group.rg_internal.name
+}
+
+data "azurerm_storage_account" "userbackups" {
+  name                = "iopstuserbackups"
+  resource_group_name = azurerm_resource_group.rg_internal.name
+}
+
+
+locals {
+  function_admin = {
+    app_settings_common = {
+      FUNCTIONS_WORKER_RUNTIME       = "node"
+      WEBSITE_NODE_DEFAULT_VERSION   = "14.16.0"
```

Comment:
```suggestion
```
from last microsoft issue we need to remove WEBSITE_NODE_DEFAULT_VERSION on linux function app on app service and use only linux_fx_version